### ISSUE TITLE
Toggle Vim Background Setting

### ISSUE THREAD

**Issue by petermbenjamin** [daehruoydeef/Yin-Yang#25]
It would be nice if Yin-Yang can toggle Vim `background` setting between `light` and `dark` as well.

As a bonus, it would be nice if Yin-Yang can toggle between light and dark Vim themes (configurable by the user based on what themes they have installed and what they prefer).

**Comment by daehruoydeef** [comment]
Do you know by any chance If vim is capable of live reloading the themes while it is running? If you have any hints on where to get started on this please share your knowledge. Seems like a really nice feature.

**Comment by petermbenjamin** [comment]
yes, you can "live reload" vim without restarting by running the command `:so /path/to/config`

**Comment by mayapony** [comment]
I am using darkman and vim-lumen to toggle vim background

just like this ⬇️

https://github.com/oskarsh/Yin-Yang/assets/62200137/babadb35-d2e6-4935-9b5e-30e3333a0986
